packages/url_launcher/README.md

```diff
@@ -67,3 +67,9 @@ best practice we suggest adjusting the application UI so that the unsupported
 URL is never triggered; for example, if the `email` scheme is not supported, a
 UI button that would have sent email can be changed to redirect the user to a
 web page using a URL following the `http` scheme.
+
+## Browser vs In-app Handling
+By default, Android opens up a browser when handling URLs. You can pass
+forceWebView: true parameter to tell the plugin to open a WebView instead. On
+iOS, the default behavior is to open all web URLs within the app. Everything
+else is redirected to the app handler.
```

Comment:
Document the forceSafariVC too for the cookie sharing use case?

Reply:
Webviews can and do share the browser context. Mentioned it in the doc.